src/System.Security.SecureString/src/System/Security/SecureString.cs

```diff
@@ -2,66 +2,23 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Diagnostics;
-using System.Runtime.InteropServices;
-using System.Security.Cryptography;
 
 namespace System.Security
 {
-    public sealed class SecureString : IDisposable
+    public sealed partial class SecureString : IDisposable
     {
-        [System.Security.SecurityCritical] // auto-generated
-        private SafeBSTRHandle _encryptedBuffer;
-
-        private int _decryptedLength;
-        private bool _readOnly;
-
         private const int MaxLength = 65536;
-
         private readonly object _methodLock = new object();
+        private bool _readOnly;
+        private int _decryptedLength;
 
         [System.Security.SecuritySafeCritical]  // auto-generated
-        static SecureString()
-        {
-        }
-
-        [System.Security.SecurityCritical]  // auto-generated
-        internal SecureString(SecureString str)
-        {
-            AllocateBuffer(str.EncryptedBufferLength);
-            SafeBSTRHandle.Copy(str._encryptedBuffer, _encryptedBuffer);
-            _decryptedLength = str._decryptedLength;
-        }
-
-        [System.Security.SecuritySafeCritical]  // auto-generated
-        public SecureString()
+        public unsafe SecureString()
         {
-            AllocateBuffer(0);
-            _decryptedLength = 0;
-        }
-
-        [System.Security.SecurityCritical]  // auto-generated
-        private unsafe void InitializeSecureString(char* value, int length)
-        {
-            _encryptedBuffer = SafeBSTRHandle.Allocate(null, 0);
-            SafeBSTRHandle decryptedBuffer = SafeBSTRHandle.Allocate(null, (uint)length);
-            _decryptedLength = length;
-
-            byte* bufferPtr = null;
-            try
-            {
-                decryptedBuffer.AcquirePointer(ref bufferPtr);
-                Buffer.MemoryCopy((byte*)value, bufferPtr, decryptedBuffer.Length * sizeof(char), length * sizeof(char));
-            }
-            finally
-            {
-                if (bufferPtr != null)
-                    decryptedBuffer.ReleasePointer();
-            }
-
-            ProtectMemory(decryptedBuffer);
+            InitializeSecureString(null, 0);
         }
 
-        [System.Security.SecurityCritical]  // auto-generated
+        [System.Security.SecurityCritical]  // auto-generatedS
```

Comment:
This one, though, actually does need to be fixed, still.

(In my head this comment has perfect relative context, but it won't seem to in either the generated email, or the viewer here.  It's amazing how those sorts of things work.)